_data/chains/eip155-192837465.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Gather Mainnet Network",
+  "chain": "GTH",
+  "network": "mainnet",
+  "rpc": [
+    "https://mainnet.gather.network"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Gather",
+    "symbol": "GTH",
+    "decimals": 18
+  },
+  "infoURL": "https://gather.network",
+  "shortName": "Gather Mainnet",
```

Comment:
Is this really a shortname?

Reply:
Please use `gth`. 
As far as I understand, this will be used as the prefix to all addresses on this chain.

Reply:
fixed